smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java

```diff
@@ -382,7 +382,7 @@ private void bindCommandsToCoDriverXboxButtons()
         this.coDriverController.rightTrigger().onTrue(
           new ParallelCommandGroup(
             new ArmToPointCommand(
-              this.subsystemCollection.getArmSubsystem(),
+              this.subsystemCollection.getArmSubsystem().andThen(new RumbleCommand()),
```

Comment:
This won't build.  Please move the .andThen down to follow the ParallelCommandGroup